File: Assignment2_1.java
//Problem Statement:-Accept one number from user and print that number of * on screen
import java.util.*;

class Assignment2_1
{
    public static void main(String Args[])
    {
        int iNo;
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the value:");
        iNo = sobj.nextInt();

        Printer pobj = new Printer();
        pobj.StarPrinter(iNo);

        sobj.close();
    }
}

class Printer
{
    public int iValue;

    Printer()
    {
        iValue = 0;
    }

    void StarPrinter(int iValue)
    {
        if(iValue < 0)
        {
            iValue = -iValue;
        }
        for(int iCnt = 0;iCnt < iValue;iCnt++)
        {
            System.out.println("*");
        }
    }
}

File: Assignment2_1.cpp
//Problem Statement:-Accept one number from user and print that number of * on screen

#include<iostream>
using namespace std;

class Printer
{
    public:
        int iNo;

        Printer()
        {
            iNo = 0;
        }

        void StarPrinter(int iNo)
        {
            if(iNo < 0)
            {
                iNo = -iNo;
            }
            for(int iCnt = 0;iCnt < iNo;iCnt++)
            {
                cout<<"*\t";
            }
        }
};
int main()
{
    int iValue = 0;

    Printer pobj;

    cout<<"Enter the Number:\n";
    cin>>iValue;
    
    pobj.StarPrinter(iValue);

    return 0;
}